cmd/zfs_object_agent/zettaobject/src/pool.rs

```diff
@@ -433,14 +444,16 @@ struct ReclaimInfo {
     reclaim_logs: Vec<ReclaimLog>,
 }
 
-impl ReclaimLog {
-    fn to_phys(&self) -> ReclaimLogPhys {
-        ReclaimLogPhys {
-            num_bits: self.num_bits,
-            prefix: self.prefix,
-            pending_frees_log: self.pending_frees_log.to_phys(),
-            pending_free_bytes: self.pending_free_bytes,
-            object_size_log: self.object_size_log.to_phys(),
+impl ReclaimInfo {
+    fn to_phys(&self) -> ReclaimInfoPhys {
+        let mut logs = Vec::new();
```

Comment:
probably not a big deal here, but it will reduce bcopy's if we can set the vec size at creation when we know how big it will be.
```suggestion
        let mut logs = Vec::with_capacity(self.reclaim_logs.len());
```

Or better yet, I think that this will do it automatically:
```rust
reclaim_logs: self.reclaim_logs.iter().map(|log| log.to_phys()).collect(),
```